base_group_backend/models/res_users.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 Pierre Verkest <pierreverkest84@gmail.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import api, models, tools
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    @api.model
+    @tools.ormcache('self.env.uid', 'group_ext_id')
```

Comment:
```suggestion
```
@AungKoKoLin1997 I think we should either backport https://github.com/odoo/odoo/pull/68111/commits/b23d5ddd3072d253102b5b7e021a92babe4a8740 to 11.0 or switch to the approach of modifying controller methods like we did in `base_auction` https://github.com/qrtl/rmm-custom/blob/15.0/base_auction/controllers/web.py.  Can you please check and suggest which way to go?

Reply:
I think we should go with current approach with backporting in odoo because current approach is more generic to granting backend access without touching many parts of the codebase.
Backport PR: https://github.com/odoo/odoo/pull/131152